Author: Gregor Riepl <onitake@gmail.com>
Description: Add VERSION definition to CuraEngine.
  The upstream CMakeFile did not pass version information to the compilation
  of main.cpp. This left the CuraEngine binary with the default version
  string of DEV. The patch fixes this minor issue.
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -141,6 +141,8 @@
 add_executable(CuraEngine src/main.cpp) #Then compile main.cpp as separate executable, and link the library to it.
 target_link_libraries(CuraEngine _CuraEngine)
 
+set_target_properties(CuraEngine PROPERTIES COMPILE_DEFINITIONS "VERSION=\"${CURA_ENGINE_VERSION}\"")
+
 # Compiling the test environment.
 if (BUILD_TESTS)
     message(STATUS "Building tests...")
